src/poliastro/twobody/orbit.py

```diff
@@ -681,6 +681,103 @@ def geostationary(
         altitude = geo_radius - attractor.R
         return cls.circular(attractor, altitude)
 
+    @classmethod
+    def heliosynchronous(
+        cls,
+        a=None,
+        ecc=None,
+        inc=None,
+        ltan=12.0,
+        argp=0 * u.deg,
+        nu=0 * u.deg,
+        epoch=J2000,
+        plane=Planes.EARTH_EQUATOR
+    ):
+        r""" Solves for a Sun-Synchronous orbit. These orbits make use of the J2
+        perturbation to precess in order to be always towards Sun. At least
+        two parameters of the set {a, ecc, inc} are needed in order to solve
+        for these kind of orbits. Relationships among them are given by:
+
+        .. math::
+            \begin{align}
+                a &= \left (\frac{-3R_{\bigoplus}J_{2}\sqrt{\mu}\cos(i)}{2\dot{\Omega}(1-e^2)^2}  \right ) ^ {\frac{2}{7}}\\
+                e &= \sqrt{1 - \sqrt{\frac{-3R_{\bigoplus}J_{2}\sqrt{\mu}cos(i)}{2a^{\frac{7}{2}}\dot{\Omega}}}}\\
+                i &= \arccos{\left ( \frac{-2a^{\frac{7}{2}}\dot{\Omega}(1-e^2)^2}{3R_{\bigoplus}J_{2}\sqrt{\mu}} \right )}\\
+            \end{align}
+
+        Parameters
+        ----------
+        a: ~astropy.units.Quantity
+            Semi-major axis.
+        ecc: ~astropy.units.Quantity
+            Eccentricity.
+        inc: ~astropy.units.Quantity
+            Inclination.
+        ltan: ~astropy.units.Quantity
+            Local time of the ascending node
+        argp : ~astropy.units.Quantity
+            Argument of the pericenter.
+        nu : ~astropy.units.Quantity
+            True anomaly.
+        epoch : ~astropy.time.Time, optional
+            Epoch, default to J2000.
+        plane : ~poliastro.frames.Planes
+            Fundamental plane of the frame.
+        """
+
+        # Constants for Sun-Synchronours Orbits (SSO)
```

Comment:
One issue would be the LTAN specification, since that is only possible for Earth. Should I special-case LTAN/LTDN then?

Reply:
Interesting, I did not thought about that... Let us wait for @Juanlu001 to review this and see how to proceed then.

Reply:
To clarify: It would be possible for other planets as well to specify a LTAN/LTDN given we have the position of the sun in the respective coordinate system & estimates for the mean anomaly of the sun so the LTAN/LTDN calculation can be based on the mean "virtual" sun.